app/javascript/components/listings/ListingsPage.js

```diff
@@ -51,6 +52,7 @@ const layout = {
 }
 
 const ListingsPage = ({ page, listings }) => {
+  Sentry.captureException(new Error('This is my fake error message with sourcemaps hopefully'))
```

Comment:
Do we want to keep this message on there or update?

Reply:
This is part of the task before merge. Once you approve, I'll remove these errors before merging